LEAF_Request_Portal/admin/templates/mod_groups.tpl

```diff
@@ -78,7 +78,7 @@
     </main>
 </div>
 <!--Loading Modal-->
-<!--{include file="../../../libs/smarty/loading_spinner.tpl" title='User Groups'}-->
+<!--{include file="../../../app/libs/smarty/loading_spinner.tpl" title='User Groups'}-->
```

Comment:
Dont use ../../../app when trying to reference somthing on the base just use the base /app because this wont work for all portals also app folder isnt the relativity for all portals/orgcharts so make sure you are using correct relativiness.

Maybe passing an APP_LIBS_PATH here would work better just be sure

Reply:
Yeah, it had the .. reference there so I didn't change it because it appears to be working the way it is. But I agree it would be better to have it referenced absolutely. I made the necessary change, appears to be working still locally.